docs/docs/50-user-guide/50-security/30-managing-credentials.md

```diff
@@ -532,6 +532,21 @@ stringData:
   repoURL: <artifact registry url>
 ```
 
+For OCI Helm chart repositories, the `Secret` should be labeled with `helm` instead:
```

Comment:
This feels like an "afterthought." I'd suggest that we rewind all the way to line 520 and introduce the two examples like:

```
To use this option, your `Secret` should take the following form:

* For a container image repository:

  ```yaml
  < example goes here>
  ```

* For an OCI Helm chart repository:

  ```yaml
  < example goes here>
  ```
```